scripts/create-tables.sql

```diff
@@ -6,8 +6,7 @@ CREATE TABLE IF NOT EXISTS "users"
   created_at timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
   updated_at timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
   deleted_at timestamp DEFAULT NULL,
-  language varchar NOT NULL DEFAULT 'en',
-  country varchar NOT NULL DEFAULT 'US',
+  language varchar NOT NULL DEFAULT 'en-US',
```

Comment:
we're going to make the modifications in the migration script, so don't change this here